src/vmspawn/vmspawn-scope.c

```diff
@@ -0,0 +1,359 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "vmspawn-scope.h"
+#include "bus-error.h"
+#include "bus-locator.h"
+#include "bus-unit-util.h"
+#include "bus-util.h"
+#include "bus-wait-for-jobs.h"
+#include "macro-fundamental.h"
+#include "macro.h"
+#include "process-util.h"
+#include "random-util.h"
+#include "sd-bus.h"
```

Comment:
sd-bus is a public header, so sort in a separate block before internal headers

Reply:
Fixed in ffc1867558.